exercises/bank-account/instructions.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Your task is to simulate a bank supporting opening/closing accounts, withdrawals, and deposits of money.
```

Comment:
This sentence reads to me like our code is responsible for managing the bank itself, but in reality, the exercise is modeling an account inside the bank.